docker/most_builder.dockerfile

```diff
@@ -11,6 +11,5 @@ RUN . "$NVM_DIR/nvm.sh" && nvm use v${NODE_VERSION}
 RUN . "$NVM_DIR/nvm.sh" && nvm alias default v${NODE_VERSION}
 ENV PATH="/root/.nvm/versions/node/v${NODE_VERSION}/bin/:${PATH}"
 
-COPY * .
-# Trigger toolchain install
-RUN cargo check || true
+COPY rust-toolchain.toml .
```

Comment:
Since you're setting the `--toolchain` manually in `typechain compile` (in `package.json`) this won't be used for contracts. It will be for relayer though.